mppm/install.md

```diff
@@ -4,10 +4,11 @@ title: Install Multiplayer Play Mode
 description: How to install Multiplayer Play Mode
 ---
 
-To begin testing your multiplayer functionality, you must first install the Multiplayer Play Mode package.
+You can use Multiplayer Play Mode in Unity Editor versions 2023.1 or later.
+To install the Multiplayer Play Mode package, do the following.
```

Comment:
Not a fan of "do the following", and there's no colon (should there be?). But that's more of a personal thing.

Reply:
Oh there should definitely be a colon, my mistake. 
As for "do the following" - that's just to make it a clause, but I can replace it with "follow these steps" as in the style guide.